src/AutoRest.CSharp/Common/Input/CodeModelPartials.cs

```diff
@@ -16,6 +15,9 @@
 // ReSharper disable once CheckNamespace
 namespace AutoRest.CSharp.Input
 {
+    [DebuggerDisplay("Languages(Name: {Default.Name})")]
```

Comment:
Not quite sure about the usage here since we can't flatten all the nested properties into debugger. We might print the important ones, but as you pointed out [here](https://github.com/Azure/autorest.csharp/pull/3761#discussion_r1333818562), the source of truth is `RequestParameter.Origin`, should we print that instead?

Reply:
never mind, it does make sense to print name here since it matches with swagger definition better.